javascript-practice/src/js/views/userView.js

```diff
@@ -0,0 +1,18 @@
+export default class UserView {
+  constructor() {
+    this.loginFormEl = document.getElementById('login-form');
+  }
+
+  bindFormLogin = (submitCallBack) => {
```

Comment:
submitCallback (not submitCallBack) is not clear what we do, consider another name.